examples/java/pom.xml

```diff
@@ -223,6 +235,29 @@
                   </systemPropertyVariables>
                 </configuration>
               </execution>
+              <execution>
+                <id>apex-runner-integration-tests</id>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <includes>
+                    <include>WordCountIT.java</include>
```

Comment:
Non-blocking: filed [BEAM-966](https://issues.apache.org/jira/browse/BEAM-966). There are matching issues for Flink runner and Spark runner.